examples/android/sync/app/src/androidTest/java/com/mongodb/realm/examples/java/ErrorHandlerTest.java

```diff
@@ -0,0 +1,44 @@
+package com.mongodb.realm.examples.java;
+import android.util.Log;
+
+import com.mongodb.realm.examples.RealmTest;
```

Comment:
Your tests are failing because you don't import the Expectation class.
```suggestion
import com.mongodb.realm.examples.RealmTest;
import com.mongodb.realm.examples.Expectation
```

Reply:
Fixed + for kotlin